test/widgets/message_list_test.dart

```diff
@@ -1188,6 +1189,77 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+
+    group('action sheet visual feedback', () {
```

Comment:
The visual feedback is also given on long-press gestures that are canceled and don't cause the action sheet to open. How about just 'background-color tint'?